Update descriptions to be less centric to the 'Become a teacher' teams (#13)
DFE-Digital/dfe-glossary: app/index.md

### Higher education institution (HEI)

> A generic term for organisations that teach at degree level or higher (’higher education’). Most of these are now called ‘universities’, which is a more commonly used term, but there are some which may call still call themselves colleges or use other terms.

Reviewer: ```suggestion
Usually a university. A generic term for organisations that teach at degree level or higher (’higher education’). Some still call themselves colleges or use another term.
```
